src/data/yellowlist.txt

```diff
@@ -99,6 +99,7 @@ www.beyondmenu.com
 bigcommerce.com
 api.bilibili.com
 www.bilibili.com
+platform.bing.com
```

Comment:
Should we add `speech.platform.bing.com` instead? To make the workaround more specific.